src/components/Editor/FreeBusy/RoomAvailabilityModal.vue

```diff
@@ -0,0 +1,361 @@
+			}, {
+				label: this.$t('calendar', 'Busy (tentative)'),
+				color: getColorForFBType('BUSY-TENTATIVE'),
+			}, {
+				label: this.$t('calendar', 'Busy'),
+				color: getColorForFBType('BUSY'),
+			}, {
+				label: this.$t('calendar', 'Out of office'),
+				color: getColorForFBType('BUSY-UNAVAILABLE'),
+			}, {
+				label: this.$t('calendar', 'Unknown'),
+				color: getColorForFBType('UNKNOWN'),
+			}]
+		},
+		options() {
+			return {
+				// Initialization:
+				initialView: 'resourceTimelineDay',
+				initialDate: this.currentStart,
+				schedulerLicenseKey: 'GPL-My-Project-Is-Open-Source',
+				// Data
+				eventSources: this.eventSources,
+				resources: this.resources,
+				// Plugins
+				plugins: this.plugins,
+				// Interaction:
+				editable: false,
+				selectable: true,
+				select: this.handleSelect,
+				// Localization:
+				...getDateFormattingConfig(),
+				...getFullCalendarLocale(),
+				// Rendering
+				height: 'auto',
+				loading: this.loading,
+				headerToolbar: false,
+				resourceAreaColumns: [
+					{
+						field: 'title',
+						headerContent: 'Room',
+					},
+				],
+				// Timezones:
+				timeZone: this.timezoneId,
+				// Formatting of the title
+				// will produce something like "Tuesday, September 18, 2018"
+				// ref https://fullcalendar.io/docs/date-formatting
+				titleFormat: {
+					month: 'long',
+					year: 'numeric',
+					day: 'numeric',
+					weekday: 'long',
+				},
+				dateClick: this.findFreeSlots(),
+			}
+		},
+	},
+	methods: {
+		closeRoomAvailability() {
+			this.showModal = false
+		},
+		handleActions(action, date = null) {
+			const calendar = this.$refs.freeBusyFullCalendar.getApi()
+			switch (action) {
+			case 'today':
+				calendar.today()
+				break
+			case 'left':
+				calendar.prev()
+				break
+			case 'right':
+				calendar.next()
+				break
+			case 'picker':
+				calendar.gotoDate(date)
+				break
+			}
+			this.currentDate = calendar.getDate()
+			calendar.scrollToTime(this.scrollTime)
+			this.findFreeSlots()
+		},
+		async findFreeSlots() {
+			// Doesn't make sense for multiple days
+			if (this.currentStart.getDate() !== this.currentEnd.getDate()) {
+				return
+			}
+
+			// Needed to update with full calendar widget changes
+			const startSearch = new Date(this.currentStart)
+			startSearch.setDate(this.currentDate.getDate())
+			startSearch.setMonth(this.currentDate.getMonth())
+			startSearch.setYear(this.currentDate.getFullYear())
+
+			const endSearch = new Date(this.currentEnd)
+			endSearch.setDate(this.currentDate.getDate())
+			endSearch.setMonth(this.currentDate.getMonth())
+			endSearch.setYear(this.currentDate.getFullYear())
+
+			try {
+				// for now search slots only in the first week days
+				const endSearchDate = new Date(startSearch)
+				endSearchDate.setDate(startSearch.getDate() + 7)
+				const eventResults = await getBusySlots(
+					this.organizer.roomProperty,
+					this.rooms.map((a) => a.roomProperty),
```

Comment:
```suggestion
				const eventResults = await getBusySlots(
					getCurrentUserPrincipal(),
					this.rooms,
```

We need to fake the organizer and use all rooms as attendees. Otherwise, the objects need to be mapped which I will implement.